release-notes/azure/v11.2.0.md

```diff
@@ -0,0 +1,69 @@
+## :zap: Giant Swarm Release 11.2.0 for Azure is now active for you! :zap:
+
+This release upgrades the NGINX Ingress Controller app to upstream ingress-nginx v0.29.0. Among multiple improvements this upgrade comes with a **breaking change** - default SSL ciphers no longer include AES-CBC based ciphers since they are considered weak. As side-effect this also drops SSL support for old browsers and clients which do not support AES-GCM ciphers (e.g. Safari 9). At your own risk weak ciphers can still be enabled on demand independently for each cluster.
+
+We also added optional metrics Service for prometheus-operator support. This allows prometheus-operator to scrape metrics to monitor the app.
+
+To ensure smooth upgrade and migration experience, and for any additional questions you may have in relation to these changes, as always, please consult with your support engineer.
+
+### azure-operator v2.9.0
+
+- No change from the previous version
```

Comment:
@cokiengchiara can you please check with other POs or somebody who may know more on platform release process, do these kind of "no change" entries make sense? I'm not sure but I think we're not consistently neither mentioning nor omitting them in release notes, across different releases and across different providers.

Reply:
Sounds good. https://gigantic.slack.com/archives/C02521EC0/p1582030174135100

I think it's a good idea. Let's add them here.

Reply:
"no change" are already here, will add them too to the other 2 release notes - those mostly have just what was changed.

Reply:
We decided to remove / not include "no change" items instead